src/sentry/static/sentry/app/views/settings/organizationMembers/inviteMember/index.jsx

```diff
@@ -203,6 +203,26 @@ const InviteMember = createReactClass({
     });
   },
 
+  handleSelectAll() {
+    let {teams} = this.getOrganization();
+    let {selectedTeams} = this.state;
+    let allSelected = teams.length === selectedTeams.size;
+
+    if (allSelected) {
+      teams.map(team => {
+        if (selectedTeams.has(team.slug)) {
+          selectedTeams.delete(team.slug);
+        }
+      });
+    } else {
+      teams.map(team => {
```

Comment:
likewise, can we just create a new set here from teams?
```suggestion
      selectedTeams = new Set(teams.map(({slug}) => slug));
```